httpx/client.py

```diff
@@ -222,6 +223,25 @@ def params(self, params: QueryParamTypes) -> None:
         timeout: typing.Union[TimeoutTypes, UnsetType] = UNSET,
         trust_env: bool = None,
     ) -> Response:
+        if cert is not None:
+            warnings.warn(
+                "Passing a 'cert' argument when making a request on a client "
+                "is due to be deprecated. Instantiate a new client instead, "
+                "passing any 'cert' arguments to the client itself."
+            )
```

Comment:
Don't forget to specify the category of warning with the second parameter:
`warn(message, category, ...)`
```suggestion
            , PendingDeprecationWarning)
```